### PR TITLE
Added guest enabled info to WhoAmI view.

### PR DESCRIPTION
**Pull request by normanjaeckel**
@emanuelschuetze Please check this out and try to control the "Continue as guest" button depending on the flag I submit to you in the who am I view with this pull request. I do not know how to do this. Maybe combine with #1762.

### CODE REVIEW

File: tests/integration/users/test_views.py [hunk 1]
```diff
@@ -12,7 +12,9 @@ def test_get_anonymous(self):
         response = self.client.get(self.url)
 
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.content, b'{"user_id":null}')
+        self.assertEqual(
+            json.loads(response.content.decode('utf8')),
```
Comment (normanjaeckel) [inline tests/integration/users/test_views.py]: Here we get the "Quittung". In Python 3.3 and 3.4 the result is `b'{"user_id":null,"guest_enabled":false}'` but in Python 3.5 it is `b'{"guest_enabled":false,"user_id":null}'` for whatever reason, so this test always failed. We should not use bytestring tests any more if we want to check JSON content.

File: tests/integration/users/test_views.py [hunk 2]
```diff
@@ -12,7 +12,9 @@ def test_get_anonymous(self):
         response = self.client.get(self.url)
 
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.content, b'{"user_id":null}')
+        self.assertEqual(
+            json.loads(response.content.decode('utf-8')),
```
Comment (normanjaeckel) [inline tests/integration/users/test_views.py]: Here we get the "Quittung". In Python 3.3 and 3.4 the result is `b'{"user_id":null,"guest_enabled":false}'` but in Python 3.5 it is `b'{"guest_enabled":false,"user_id":null}'` for whatever reason, so this test always failed. We should not use bytestring tests any more if we want to check JSON content.

Comment (ostcar) [inline tests/integration/users/test_views.py]: ok. But you can remove the string 'utf-8'. It is the default value of the argument: https://docs.python.org/3.5/library/stdtypes.html#bytes.decode

**Comment by emanuelschuetze** [comment]
see #1765.

**Comment by normanjaeckel** [comment]
Closed with respect of #1765.

**Pull request closed without merging**